.gitignore

```diff
@@ -1 +1,20 @@
-flagged
```

Comment:
Flagged is the history of the chats, I kept it on my computer, should I add an explanation about it in the README file?

Reply:
I didn't see a flagged being created for me. Strange. 

Anyway, yes, should create a very small PR which does something like this (would be a good excersie) -
```
# pyenv
.python-version

# Other?!
flagged
```
Changed to -
```
# pyenv
.python-version

# Chat-bot artifacts/history
flagged

# Drop here "other" stuff as you wish..
```